.drone.yml

```diff
@@ -154,6 +156,20 @@ services:
   image: redis
   ports:
     - 6379
+- name: protectedredis
+  image: redis
+  ports:
+    - 6380
+  volumes:
+    - name: RedisConf
+      path: "/usr/local/etc/redis/redis.conf"
+  commands:
+    - "redis-server /usr/local/etc/redis/redis.conf"
+volumes:
```

Comment:
this needs to be added to be moved to the bottom because now `athens-proxy` is being considered as part of the volume :)

Reply:
Yeah its very strange to be honest.... I'm just testing some stuff since the `drone` documentation is meh when it comes to mounting `host volumes`.....

Reply:
But it also doesn't like `./test` it wants a full path....

Reply:
It wouldn't work anyway given their highlighted doc: 

>This setting is only available to trusted repositories, since mounting host machine volumes is a security risk.

Also note that each image in the pipeline gets its own volume where the WORKDIR is the repository root dir, so you can just immediately use that without having to volume anything, as far as i know.

Reply:
This i didn't know... I don't think they really have that documented anywhere when looking at volume or command stuff :/

Reply:
@mplachter no worries, let me know when you're all set and I'll merge. Thanks for all the updates!